project/Dependencies.scala

```diff
@@ -78,6 +80,7 @@ object Dependencies {
   }
 
   object Protobuf {
+    val protobufJava = "com.google.protobuf" % "protobuf-java" % Versions.googleProtobuf
```

Comment:
So scalapb-runtime actually pulls in the older protobuf-java artifact with generated classes as well as protoc, if that doesn't match up with the protoc version things fail.